src/Psalm/Report/ConsoleReport.php

```diff
@@ -102,4 +106,21 @@ private function getTaintSnippets(array $taint_trace) : string
 
         return $snippets;
     }
+
+    /**
+     * @param IssueData|DataFlowNodeData $data
+     */
+    private function getFileReference($data): string
+    {
+        if (null === $this->linkFormat) {
+            // if xdebug is not enabled, use `get_cfg_var` to get the value directly from php.ini
+            $this->linkFormat = ini_get('xdebug.file_link_format') ?: get_cfg_var('xdebug.file_link_format')
+                ?: 'file://%f#L%l';
+        }
+
+        $link = strtr($this->linkFormat, ['%f' => $data->file_path, '%l' => $data->line_from]);
+        $reference = $data->file_name . ':' . $data->line_from . ':' . $data->column_from;
+
+        return "\033]8;;" . $link . "\033\\" . $reference . "\033]8;;\033\\";
```

Comment:
Perhaps we should disable links when Psalm is being run with '-m' option (you can check it with `$this->use_color`) as that's the only option users have to prevent buggy terminals from displaying ANSI sequences.